engine/schema/src/main/java/com/cloud/storage/VMTemplateVO.java

```diff
@@ -230,6 +230,13 @@ public static VMTemplateVO createPreHostIso(Long id, String uniqueName, String n
         return template;
     }
 
+    public static VMTemplateVO createSystemIso(Long id, String uniqueName, String name, boolean isPublic,
```

Comment:
Import of  VM is allowed without using any template. But since CS needs a template or ISO ID for a VM, we need to create a dummy, removed ISO which can be used for import in such cases. Method to create dummy ISO DB record, https://github.com/apache/cloudstack/pull/3606/files#diff-3eaaf9de1645091dff197645123e669cR200

Reply:
ok, unfortunately I am talking functionality again, but... I think the current state (snapshot'd) is the best solution for template for this VM in such cases, unless there is a common template known that several imported VMs can be referring.

Reply:
This dummy ISO will be created only once and will be used for all imported VMs thereafter when no template is passed in the API call (will be common for all imported VMs).